app/test/hybridDefaults.test.ts

```
@@ -0,0 +1,20 @@
+import fs from 'fs';
+import path from 'path';
+
+const src = fs.readFileSync(
+  path.join(__dirname, '../src/services/HybridRecognizer.ts'),
+  'utf8',
+);
+const match = src.match(/HYBRID_DEFAULTS\s*=\s*({[\s\S]*?})/);
+// eslint-disable-next-line no-eval
+const HYBRID_DEFAULTS = match ? eval('(' + match[1] + ')') : {};
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)